src/main/resources/localization/en.conf

```diff
@@ -124,3 +124,16 @@ CommonAccess: "Common"
   "%s fell onto his side and is looking around himself with eyes full of pain and suffering.",
   "%s is barely breathing, letting out husky sounds and coughing up bloody foam."
 ]
+
+// Command usage:
+"command.help.bash": "Usage: $bash: A random quote from http://bash.im."
+"command.help.dice": "Usage: $dice [faces] [count]: roll a [count] of [faces]-sided dice."
+"command.help.help": "Usage: $help: display available commands. | $help [command]: dislay command description."
+"command.help.loglist": "Usage: $loglist [quoteId]: a quote from http://loglist.net. Gives a random quote if no quoteId is given."
+"command.help.pet": "See $pet help."
+"command.help.s": "Usage: $s |$s [number]: shorthand for $say"
```

Comment:
The funny thing is that I've known about the replace command, but seeing it in Markov plugin made me have a brain fart -_\